core-common/src/main/java/org/mapstruct/IterableMapping.java

```diff
@@ -47,6 +47,14 @@
     String dateFormat() default "";
 
     /**
+     * A format string as processable by {@link java.text.DecimalFormat} if the annotated method maps from a
```

Comment:
didn't even know that was possible (having imports just for the sake of javadoc and no compiler warnings) 👍